scripts/ocr_runner.py

```
@@ -0,0 +1,198 @@
+            try:
+                obj = json.loads(line)
+                if 'sha256' in obj:
+                    m[obj['sha256']] = obj
+            except Exception:
+                continue
+    return m
+
+
+def pdftotext_count(path):
+    try:
+        p = subprocess.run(['pdftotext', '-layout', '-enc', 'UTF-8', str(path), '-'], stdout=subprocess.PIPE, stderr=subprocess.DEVNULL, check=False, timeout=60)
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)